Cds 21 multibutton fast review

beawar opened the pull request.

beawar committed:
refactor(MultiButton): move to typescript and use Button internally

refs: CDS-21

beawar committed:
fix(Dropdown): respect forceOpen when closing dropdown

When dropdown detect that an action that should lead to the closing of the popper,
set it's state to the forceOpen prop, in order to allow a controlled usage of the component

beawar committed:
feat(Button): allow set of ref for secondary action button

refs: CDS-21

beawar committed:
chore: update eslint configs for ide

CataldoMazzilli commented on src/components/inputs/MultiButton.md:
````diff
@@ -4,21 +4,166 @@ SPDX-FileCopyrightText: 2021 Zextras <https://www.zextras.com>
+
+```
+### Width
+
````
width prop fill does not work: probably due to the dropdown

beawar committed:
fix(MultiButton): make multibutton respect width prop

- when width is set to fill, set external container to 100% width
- allow customization of dropdown list width through "dropdownWidth" prop

refs: CDS-21
